<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseModuleAutoTile</class>
 <widget class="QWidget" name="DatabaseModuleAutoTile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>196</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>auto-tile-set:</string>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pTopLeft">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;top-left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pCenterLeft">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;center-left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pBottomRight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;bottom-right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pCenterRight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;center-right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pInnerEdges">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;inner edges&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pBottomCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;bottom-center&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>basic set:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>inner edges:</string>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pTopCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;top-center&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;center&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pTopRight">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;top-right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TileDropLabel" name="m_pBottomLeft">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="test">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;bottom-left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileDropLabel</class>
   <extends>QLabel</extends>
   <header>DatabaseModuleDragDrop.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
